mappings/net/minecraft/nbt/NbtElement.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS net/minecraft/class_2520 net/minecraft/nbt/NbtElement
+	COMMENT Represents an NBT element.
+	COMMENT <p>
+	COMMENT Each type of NBT element has an associated integer {@linkplain NbtElement#getType NBT type}. The following is a list of possible types:
+	COMMENT <ul>
+	COMMENT <li>{@code 0} - null</li>
+	COMMENT <li>{@code 1} - byte</li>
+	COMMENT <li>{@code 2} - short</li>
+	COMMENT <li>{@code 3} - int</li>
+	COMMENT <li>{@code 4} - long</li>
+	COMMENT <li>{@code 5} - float</li>
+	COMMENT <li>{@code 6} - double</li>
+	COMMENT <li>{@code 7} - byte array</li>
+	COMMENT <li>{@code 8} - string</li>
+	COMMENT <li>{@code 9} - list</li>
+	COMMENT <li>{@code 10} - compound object</li>
+	COMMENT <li>{@code 11} - int array</li>
+	COMMENT <li>{@code 12} - long array</li>
+	COMMENT <li>{@code 99} - only used for checking the existence of a number element in a {@linkplain NbtCompound NBT compound object}.</li>
```

Comment:
This should probably be moved out of the list if it is never returned by `NbtElement#getType`.

Reply:
The line was removed completely in 223dd3a6791f9954a7fc92a16108cd787f5cdfb8, but I think that it would be better if it was just an additional note at the end.